.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm run pretty-fix
```

Comment:
`pre-commit` hook must only `check` your code
You should run here `eslint check` script and `prettier check` script

The problem is that in this variant it will work like:
1. git add .
2. git commit -m "message"
3. *pre-commit runs and changes files because of --fix option*
4. you cannot commit before you run git add . again (because changed files ar e not staged in git

Reply:
Thank you very much for code review!

Reply:
yep, fix it